src/test/java/org/springframework/data/redis/Address.java

```diff
@@ -91,9 +91,8 @@ public boolean equals(@Nullable Object obj) {
 			return true;
 		if (obj == null)
 			return false;
-		if (!(obj instanceof Address))
+		if (!(obj instanceof Address other))
```

Comment:
In `equals` methods, I generally trend towards the name of the object, such as `address` or use `that`.

Reply:
Changed to `that`.